### PR TITLE
docs(zh): translate routing

### PR DESCRIPTION
**Pull request by kecrily**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [ ] New or updated content
- [x] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description

translate routing

### CODE REVIEW

File: src/pages/zh-cn/core-concepts/routing.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+---
+// Example: src/pages/posts/[id].astro
+const { id } = Astro.params;
+---
+<p>Post: { id }</p>
+
+
+// Astro.params object passed for the route `/post/abc`
+{ "id": "abc" }
+```
```
Comment (Yan-Thomas) [inline src/pages/zh-cn/core-concepts/routing.md:84]: There is some missing translations here

File: src/pages/zh-cn/core-concepts/routing.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+- 没有路径参数的静态路由将优先于所有其他路由
+- 使用命名参数的动态路由优先于其余参数
+- 其他参数的优先级最低
+- 平局则按字母顺序解决
+
+鉴于上面的例子，下面是几个例子，说明规则如何将请求的URL与用于建立HTML的路由相匹配。
+
+- `pages/posts/create.astro` - Will build `/posts/create`
+- `pages/posts/[pid].astro` - Will build `/posts/1`, `/posts/abc`, etc. But not `/posts/create`
+- `pages/posts/[...slug].astro` - Will build `/posts/1/2`, `/posts/a/b/c`, etc. But not `/posts/create`, `/posts/1`, `/posts/abc`
```
Comment (Yan-Thomas) [inline src/pages/zh-cn/core-concepts/routing.md:161]: Here too

File: src/pages/zh-cn/core-concepts/routing.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+<h1>Page {page.currentPage}</h1>
+<ul>
+  <!-- 列出宇航员信息数组 -->
+  {page.data.map(({ astronaut }) => <li>{astronaut}</li>)}
+</ul>
+```
+
+将生成以下两个页面：
+- `/astronauts/1` - Page 1: Displays "Neil Armstrong" and "Buzz Aldrin"
+- `/astronauts/2` - Page 2: Displays "Sally Ride" and "John Glenn"
```
Comment (Yan-Thomas) [inline src/pages/zh-cn/core-concepts/routing.md:201]: And here as well.

**Review by Yan-Thomas — COMMENTED** [review]
Hi @kecrily, thanks for the PR!! I pointed out some stuff that I think you forgot to translate. I'm not sure what is breaking the build of your PR though, I'll try to update the branch and see if it magically fixes it.

File: src/pages/zh-cn/core-concepts/routing.md [hunk 4]
```diff
@@ -0,0 +1,291 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: 路由
+description: 关于 Astro 路由的介绍。
+
```
Comment (delucis) [inline src/pages/zh-cn/core-concepts/routing.md:5]: I think this breaking the build: missing the closing tag for the frontmatter:

```suggestion
---

```

**Comment by kecrily** [comment]
Let me check it

**Review by delucis — APPROVED** [review]
Perfect. Thank you!

**Pull request merged**